React/Views/RCTView.h

```diff
@@ -79,6 +80,11 @@ extern const UIAccessibilityTraits SwitchAccessibilityTrait;
 @property (nonatomic, assign) CGFloat borderBottomStartRadius;
 @property (nonatomic, assign) CGFloat borderBottomEndRadius;
 
+/**
+ * Border curve
+ */ 
+@property (nonatomic, assign) RCTBorderCurve borderCurve;
```

Comment:
I don't think we need that prop since we're setting the value on view.layer directly in the view manager.